tests/test_flows.py

```diff
@@ -365,10 +365,12 @@ def test_ap_capability_query(self):
 
         debug("Confirming ap capability query has been received on agent")
         self.check_log(env.agents[0], "AP_CAPABILITY_QUERY_MESSAGE")
-
+        capability_query = env.wired_sniffer.get_packets_for_cmdu_type(self.running,0x8001)
```

Comment:
The example should replace the check_log, not just add a call without testing it. Also I think you should add an API to search for specific packets like it is done in the UCC.

For example, this is how the UCC is checking for a specific topology response:

```
2020-04-14 04:08:41.506 - INFO - 
        ~~~~~ 4.1.1 Step 9 sniffer checks - Topology Response message ~~~~~

2020-04-14 04:08:41.512 - INFO - SNIFFER (192.168.250.5:9999) ---> sniffer_control_filter_capture,InFile,_MAP-4.1.1-STEP9,OutFile,MAP-4.1.1-STEP9,msgId,0x43c6,filterfile,mapfile,msgType,0x0003,nframes,1
2020-04-14 04:08:42.023 - INFO - SNIFFER (192.168.250.5:9999) <-- status,COMPLETE,FilterStatus,SUCCESS,FrameCount,1

```

I think the equivalent in python should be something like:

```
topology_response = env.wired_sniffer.get_packets(self.running, cmdu_type=0x0003, mid=0x43c6)
```
The API should also allow for more arguments - source and destination MACs so we can filter on ALid, and probably more that I can't think of now. Every parameter should be optional except for self.running.